Lesson7/feyza/Lesson4-insight/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Insight</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="navbar">
```

Comment:
Change the content of the <nav> element to be as below so that the logo and other list items can be located on opposite sides of the screen:

```
<a href="./" class="logo">Insight</a>
      <ul>
        <!-- <li class="logo"><a href="./">Insight</a></li> -->
        <!--a etiketine değer verince hepsi etkileniyor logoyu ayırıp düzeltemedim-->
        <li class="nav-list"><a href="#">Overview</a></li>
        <li class="nav-list"><a href="#">Pages</a></li>
        <li class="nav-list"><a href="#">Blocks</a></li>
        <li class="nav-list"><a href="#">Components</a></li>
      </ul>
```